catalog/opensearch/catalog-opensearch-endpoint/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -41,4 +41,18 @@
         <argument ref="filterBuilder"/>
     </bean>
 
+    <bean id="ddf.catalog.endpoint.opensearch"
```

Comment:
Looks like we are duplicating information (that can get out of sync) between the jaxrs:server definition and the CatalogEndpoint definition.  Also there's nothing to enforce a developer to create and register this bean.  Should this be done for all endpoints?